### ISSUE TITLE
`http status 404` when trying to install a modpack or a server

### ISSUE THREAD

**Issue by henkelmax** [nothub/mrpack-install#54]
Version: `v0.17.3-beta`
OS: Win 11

When I try to install a neoforge modpack using `mrpack-install .\mypack.mrpack` I get

```
Server file not present, downloading...
(Point --server-dir and --server-file to existing targets to skip this step)
http status 404
```

Trying `mrpack-install server neoforge` or `mrpack-install server neoforge --flavor-version 21.1.65 -v` or even `mrpack-install server forge`  results in
```
http status 404
```

**Comment by nothub** [comment]
Hi, thanks for the report!

I can confirm the issue, seems like the neoforged maven url structure changed.

**Comment by nothub** [comment]
The problem was fixed in new release [`v0.18.0-beta`](https://github.com/nothub/mrpack-install/releases/tag/v0.18.0-beta).
Thanks for reporting the issue!

**Comment by nothub** [comment]
The forge problem was also fixed now :see_no_evil:
For that I gonna release a new version [`v0.18.1-beta`](https://github.com/nothub/mrpack-install/releases/tag/v0.18.1-beta) in a second.
